content/en/docs/concepts/storage/volumes.md

```diff
@@ -1061,7 +1061,7 @@ persistent volume:
   call to the CSI driver. In order to use the `nodeExpandSecretRef` field, your
   cluster should be running Kubernetes version 1.25 or later.
 * If you are running Kubernetes Version 1.25 or 1.26, you must enable
```

Comment:
We can drop guidance aimed at people running unsupported releases, and I typically recommend that we do.

Reply:
removed.